stellenbosch-law-review.csl

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-US">
```

Comment:
> default-locale="en-US"

Should we remove the default-locale? Per http://blogs.sun.ac.za/iplaw/files/2018/02/Stellenbosch-Law-Review-Style-Guide.pdf, "The Stellenbosch  Law  Reviewis  published  tri-annually  and  contains  contributions  in  English  or Afrikaans."